tests/scale-tests/harness/harnessctl/kube_apply.go

```
@@ -0,0 +1,161 @@
+
+// applyYAMLURL fetches a remote manifest URL and applies it.
+func (c *clients) applyYAMLURL(ctx context.Context, url string) error {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
+	if err != nil {
+		return err
+	}
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return fmt.Errorf("fetch %s: %w", url, err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)